docs/configuration/examples.md

```diff
@@ -118,6 +118,23 @@ changelog-entry:
 
 </details>
 
+### Creating source archive using 'tito' during release syncing
+<details>
+  <summary>Using pre-sync for running tito</summary>
+
+```yaml
+actions:
+  pre-sync:
+    - tito build -o . --tgz
+
+files_to_sync:
+  - src:
```

Comment:
It won't be committed, will it?

Reply:
no it shouldn't, I took this from existing working configs